### PR TITLE
Create Cluster Configuration Repository (CCR) for munnin.

### PR DESCRIPTION
**Pull request by davidewatson**

### CODE REVIEW

File: configs/pipeline-munnin.cluster.cnct.io.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+type: cnct
+uniqueId: pipeline-munnin.cluster.cnct.io
```
Comment (zachpuck) [inline configs/pipeline-munnin.cluster.cnct.io.yaml]: typo on name. should be "muninn"

File: configs/pipeline-munnin.cluster.cnct.io.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+displayName: muninn.cluster.cnct.io
+description: Cluster Configuration Repository
+apiUrl: "https://api.github.com"
+org: samsung-cnct
+repo: munnin.cluster.cnct.io
```
Comment (zachpuck) [inline configs/pipeline-munnin.cluster.cnct.io.yaml]: should be "muninn"

File: configs/pipeline-munnin.cluster.cnct.io.yaml (hunk 1 above)
Comment (davidewatson) [reply configs/pipeline-munnin.cluster.cnct.io.yaml]: Good catch, thanks!

File: configs/pipeline-munnin.cluster.cnct.io.yaml (hunk 2 above)
Comment (davidewatson) [reply configs/pipeline-munnin.cluster.cnct.io.yaml]: I predict this misspelling will not be uncommon... ;)

Comment (philoserf) [reply configs/pipeline-munnin.cluster.cnct.io.yaml]: Spelling is so 18th and 19th century.

**Pull request merged**